packages/playwright-core/src/client/playwright.ts

```diff
@@ -82,7 +83,11 @@ export class Playwright extends ChannelOwner<channels.PlaywrightChannel> {
   }
 
   _allContexts() {
-    return this._browserTypes().flatMap(type => [...type._contexts]);
+    return [
+      ...this._browserTypes().flatMap(type => [...type._contexts]),
+      ...this._electron._contexts,
```

Comment:
I do not think either electron or android were ever supported in our fixtures, so let's not add them.

Reply:
What about renaming `_allContexts` to `_allNativeContexts` (or different if you have suggestions) and having `_allContexts` return electron/android as well? I see the confusion that `_allContexts` is used inside `ArtifactsRecorder` as well as `Selectors` class.